copyright

```diff
@@ -270,6 +270,11 @@ Copyright: Various
 License: public-domain
  Taken from unsplash.com, a collection of photographs that have been donated and
  placed in the public domain.
+ 
+ Files: images/outfit/skylance*
+Copyright: Evan Fluharty (Evanfluharty@gmail.com)
+License: CC-BY-SA-4.0
+Comment: Derived and completed from works by Maximilian Korber(Wrzlprnft), Fernando S. Sáez(Azure Nyoki), and originally drawn up by bladewood (info needed)(all from under the same license)
```

Comment:
There should be spaces between the full names and the usernames (and I'm not sure if we should actually put the latter).
I don't think diacritics are allowed here, though this is a copyright file, not a data file, so I'm not certain.
My name is `Tommy Thach`.